client/lcd/test_helpers.go

```diff
@@ -337,7 +337,7 @@ func InitializeTestLCD(t *testing.T, nValidators int, initAddrs []sdk.AccAddress
 
 	cleanup = func() {
 		logger.Debug("cleaning up LCD initialization")
-		node.Stop()
+		node.Stop() //nolint:errcheck
```

Comment:
do we still have `nolint`??

Reply:
oh yes, we do https://github.com/golangci/golangci-lint#false-positives